pkg/gossip/state/stateupdate.go

```diff
@@ -77,3 +137,67 @@ func (h *UpdateHandler) handleStateUpdate(channelID string, chaincodeName string
 
 	return nil
 }
+
+func (h *UpdateHandler) getCacheUpdater(channelID string) cacheUpdater {
+	cu, err := h.cacheUpdaters.Get(channelID)
+	if err != nil {
+		// Should never happen
+		panic(err)
```

Comment:
same as above

Reply:
An error from gcache only occurs if you use a custom serializer and the serializer returns an error. Since we don't use a serializer this will never happen.